source/features/raw-view-link.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import features from '../libs/features';
+
+const childList = {childList: true};
+
+const createRawUrl = (href: string): string => {
+	const url = href.split('/');
+	url[3] = 'raw'; // Replaces 'blob'
+	return url.join('/');
+};
+
+const observer = new MutationObserver(([{addedNodes}]) => {
+	for (const node of addedNodes) {
+		const element = node as Element;
+		if (element.tagName === 'DIV') {
+			if (element.className === 'js-diff-progressive-container') {
+				observer.observe(element, childList);
+			} else {
+				addRawBtn(element);
+			}
+		}
+	}
+});
+
+function addRawBtn(node: Element) {
+	for (const link of select.all('.file-header .file-actions > div.BtnGroup [href*=\'/blob/\']', node)) {
+		link.after(
+			<a href={createRawUrl(link.getAttribute('href'))} class="btn btn-sm BtnGroup-item">
```

Comment:
I wasn't able to get the `pathname` property from a raw Element. I need to cast it to an `HTMLAnchorElement`. Also, I didn't face any problem using the href because the path is always relative.
Does this look ok to you `createRawUrl((link as HTMLAnchorElement).pathname)`?

Reply:
You can use `select.all<HTMLAnchorElement>` and `link` will automatically be one